### PR TITLE
Add installation instructions for mongodb 3.x

### PR DESCRIPTION
**Pull request by goosemania**
closes #4228
https://pulp.plan.io/issues/4228

### CODE REVIEW

File: docs/user-guide/installation/fedora.rst [hunk 1]
```diff
@@ -20,10 +20,15 @@ MongoDB replica sets if you'd like to have higher availability.
 
    $ sudo dnf install mongodb-server
 
-You need mongodb-server with version >= 2.4 installed for Pulp server. It is highly recommended
-that you `configure MongoDB to use SSL`_. If you are using Mongo's authorization feature, you
-will need to grant the ``readWrite`` and ``dbAdmin`` roles to the user you provision for Pulp to
-use. The ``dbAdmin`` role allows Pulp to create collections and install indices on them.
+You need mongodb-server with version >= 2.4 installed for Pulp server. MongoDB 2.x reached its EOL.
```
Comment (daviddavis) [inline docs/user-guide/installation/fedora.rst]: Would it be possible to require mongodb 3.x?

**Comment by daviddavis** [comment]
Thanks for updating this.

File: docs/user-guide/installation/fedora.rst (hunk 1 above)
Comment (goosemania) [reply docs/user-guide/installation/fedora.rst]: https://docs.mongodb.com/ecosystem/drivers/driver-compatibility-reference/#python-driver-compatibility

and pymongo as well :(

Comment (daviddavis) [reply docs/user-guide/installation/fedora.rst]: Cool, I have [a task open](https://pulp.plan.io/issues/3259) where we can maybe figure out how to drop support for MongoDB 2. We can continue the conversation there.

**Comment by goosemania** [comment]
@daviddavis , maybe it's better to refer to SCLs, what do yuo think? https://www.softwarecollections.org/en/scls/rhscl/rh-mongodb34/

At least Katello requires this one ^

**Comment by daviddavis** [comment]
After talking with the build team, I think SCLs are the way to go. Also, they recommended dropping support for mongo 2 as well.

**Comment by goosemania** [comment]
Detailed instructions to run mongodb from SCLs are needed. After IRC discussion, we decided to merge it as-is for now. We'll add all the details for SCLs when Mongo 2.x support is dropped in Pulp

**Pull request merged**